### PR TITLE
feat: Colonne des trajets incités

### PR DESCRIPTION
**Pull request by jonathanfallon**
Ajout de la colonne des trajets incités dans l'onglet "Tranches" des APDF.
Améliorations du layout du XLSX.

### CODE REVIEW

**Comment by jonathanfallon** [comment]
# ☁️ Nx Cloud Report

CI is running/has finished running commands for commit 6138c69587215803b24a07fc741c75708fcd4813. As they complete they will appear below. Click to see the status, the terminal output, and the build insights.

[📂 See all runs for this branch](https://cloud.nx.app/branch?workspaceId=63ebd539a53ecd331737f524&branchName=2033,feat/apdf-layout-improvements)

---


<details>
<summary>✅ Successfully ran 1 target </summary>

- [`lerna run build`](https://cloud.nx.app/runs/WvBKlznnrG)
</details>

---

Sent with 💌 from [NxCloud.](https://nx.app)

**Comment by jonathanfallon** [comment]
# ☁️ Nx Cloud Report

CI is running/has finished running commands for commit bf1564007ab1570abeea613b93ae5ca0f3dc869c. As they complete they will appear below. Click to see the status, the terminal output, and the build insights.

[📂 See all runs for this branch](https://cloud.nx.app/branch?workspaceId=63ebd539a53ecd331737f524&branchName=2033,feat/apdf-layout-improvements)

---

| 🟥 Failed Commands | 
| --- | 
| [`lerna run test:integration`](https://cloud.nx.app/runs/3qJCUEOyIr) | 

<details>
<summary>✅ Successfully ran 2 targets </summary>

- [`lerna run build`](https://cloud.nx.app/runs/cz4t4TVg8f)
- [`lerna run test:unit`](https://cloud.nx.app/runs/Ru7VOXaLnh)
</details>

---

Sent with 💌 from [NxCloud.](https://nx.app)

File: api/services/apdf/src/providers/excel/BuildExcel.ts [hunk 1]
```diff
@@ -5,21 +5,21 @@ import { CampaignSearchParamsInterface } from '../../interfaces/APDFRepositoryPr
...
-import { DataWorkBookWriter } from './writer/DataWorkbookWriter';
-import { SlicesWorkbookWriter } from './writer/SlicesWorkbookWriter';
 
 @provider()
 export class BuildExcel {
   constructor(
     private apdfRepoProvider: DataRepositoryProvider,
-    private dataWorkbookWriter: DataWorkBookWriter,
-    private slicesWorkbookWriter: SlicesWorkbookWriter,
+    private TripsWsWriter: TripsWorksheetWriter,
```
Comment (P3rceval) [inline api/services/apdf/src/providers/excel/BuildExcel.ts:16]:
```suggestion
    private tripsWsWriter: TripsWorksheetWriter,
```

File: api/services/apdf/src/providers/excel/BuildExcel.ts [hunk 2]
```diff
@@ -51,19 +51,19 @@ export class BuildExcel {
...
+    await wbWriter.commit();
 
     return { filename, filepath };
   }
 
   private async writeTrips(wkw: stream.xlsx.WorkbookWriter, params: CampaignSearchParamsInterface): Promise<void> {
     try {
       const tripCursor = await this.apdfRepoProvider.getPolicyCursor(params);
-      await this.dataWorkbookWriter.call(tripCursor, wkw);
+      await this.TripsWsWriter.call(tripCursor, wkw);
```
Comment (P3rceval) [inline api/services/apdf/src/providers/excel/BuildExcel.ts:66]:
```suggestion
      await this.tripsWsWriter.call(tripCursor, wkw);
```

**Review by P3rceval — CHANGES_REQUESTED** [review]

**Review by P3rceval — APPROVED** [review]

**Pull request merged**